CMakePresets.json

```diff
@@ -0,0 +1,193 @@
+﻿{
+  "version": 3,
+  "configurePresets": [
+    {
+      "name": "superbuild-base",
+      "hidden": true,
+      "cmakeExecutable": "cmake",
+      "generator": "Ninja",
+      "binaryDir": "${sourceDir}/out/${hostSystemName}-build/${presetName}",
```

Comment:
sequoia does implicitly assume that certain things are in projectRoot/build.

In principle I could change this but I want to make sure it's necessary!

Reply:
If this is the case it should be using `PROJECT_BINARY_DIR` as the path is command-line configurable as per this being an example for cross-build